lib/assessment.rb

```diff
@@ -13,28 +16,31 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    (5..10).inject { |sum, n| sum + n }
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
   def some_odd(numbers)
-    # Your code goes here
-  end
+    a = [7, 5, 9, 2, 0, 7, 9, 4, 2, 0]
+    p a.slice_when {|i, j| i.even? != j.even? }.to_a
+
+    end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
-  def every_even(numbers)
-    # Your code goes here
-  end
+    [1,2,3,4,5].select { |num|  num.even?  }
```

Comment:
Your test is correct, but your enumerable method is not. Also, where is the method definition? We want to be able to call `every_even` on any given set of numbers and get back either `true` or `false`